lib/collections/schemas/products.js

```diff
@@ -276,6 +318,19 @@ export const ProductVariant = new SimpleSchema({
   workflow: {
     type: Workflow,
     optional: true
+  },
+
+  originCountry: {
```

Comment:
originCountry should not be required.

![basic_reaction_product](https://cloud.githubusercontent.com/assets/72819/23344151/48459266-fcb2-11e6-9cb5-bd44f212f6e8.png)